messaging/src/main/java/org/axonframework/queryhandling/duplication/DuplicateQueryHandlerResolution.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2010-2019. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.queryhandling.duplication;
+
+/**
+ * Enumeration describing a set of reasonable {@link DuplicateQueryHandlerResolver} implementations. Can be used to
+ * configure how a {@link org.axonframework.queryhandling.QueryBus} should react upon a duplicate subscription of a
+ * query handling function.
+ *
+ * @author Mitchell Herrijgers
+ * @since 4.6
+ */
+public abstract class DuplicateQueryHandlerResolution {
+
```

Comment:
I spot I missed this on the `DuplicateCommandHandlerResolution` (...sadly enough), but as this is a utility class, I don't expect users to construct an instance of the `DuplicateQueryHandlerResolution` ever.
Hence, a private constructor would be good to add here.

Reply:
I added it, and on the missed one as well. My IDE (or sonar) recommends making an interface out of it, we could also do that